src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -101,23 +115,46 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String conversationTitle = request.getParameter("conversationTitle");
     if (!conversationTitle.matches("[\\w*]*")) {
-      request.setAttribute("error", "Please enter only letters and numbers.");
-      request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
-      return;
+      if (conversationTitle.contains("#")) {
+        String hashtagTitle = (String)conversationTitle.substring(conversationTitle.indexOf("#") + 1);
+        if (hashtagStore.isTitleTaken(hashtagTitle) == false) {
+          Hashtag hashtag = new Hashtag(UUID.randomUUID(), user.getId(), hashtagTitle, Instant.now());
```

Comment:
Here the hashtag object is created but I think it is not added to the HashtagStore

Reply:
+1